BookPlayer/Player/ChaptersViewController.swift

```diff
@@ -11,7 +11,7 @@ import MediaPlayer
 import Themeable
 import UIKit
 
-class ChaptersViewController: UITableViewController, TelemetryProtocol {
+class ChaptersViewController: UITableViewController {
```

Comment:
```suggestion
final class ChaptersViewController: UITableViewController {
```
To avoid dynamic dispatch